su-template/default/module/html/method_summary.erb

```diff
@@ -0,0 +1,16 @@
+<% if method_listing.size > 0 %>
+  <% groups(method_listing) do |list, name| %>
+    <h2>
+      <a name="<%= name.gsub(' ', '_').downcase %>">
```

Comment:
Ah, I see the visual change now. The text get styled as a link, even though the `href` attribute is missing.

We can move the anchor to the `h2` itself. `<h2 id="<%= name.gsub(' ', '_').downcase %>">`

I should have seen that.